.experimenter.yaml

```
@@ -58,6 +58,14 @@ nimbus-validation:
     settings-title:
       type: string
       description: The title of displayed in the Settings screen and app menu.
+onboardingcreen:
```

```suggestion
onboardingcreen:
```
Similar comment below